rllib/agents/ddpg/ddpg.py

```diff
@@ -88,19 +88,22 @@
         "explore": False
     },
     # === Replay buffer ===
-    # Size of the replay buffer. Note that if async_updates is set, then
-    # each worker will have a replay buffer of this size.
-    "buffer_size": DEPRECATED_VALUE,
     "replay_buffer_config": {
-        "_enable_replay_buffer_api": True,
         "type": "MultiAgentPrioritizedReplayBuffer",
         "capacity": 50000,
+        # Specify prioritized replay by supplying a buffer type that supports
+        # prioritization
```

Comment:
... ", for example: MultiAgentPrioritizedReplayBuffer".

Reply:
It seems like this is being deprecated anyways